eox_tagging/models.py

```diff
@@ -33,7 +35,7 @@ def create_tag(self, **kwargs):
         """Method used to create tags."""
         target = kwargs.pop('target_object', None)
         if target and target.__class__.__name__ in OPAQUE_KEY_PROXY_MODEL_TARGETS:
-            kwargs['target_object'] = OpaqueKeyProxyModel.objects.create(opaque_key=target.course_id)
+            kwargs['target_object'], _ = OpaqueKeyProxyModel.objects.get_or_create(opaque_key=target.course_id)
```

Comment:
don't mix double quotes with simple quotes in the same file

Reply:
Fixed! Thank you